src/components/ModalEmailSubscription.vue

```diff
@@ -0,0 +1,76 @@
+<script setup lang="ts">
+const props = defineProps<{
+  open: boolean;
+  address: string;
+}>();
+
+const emit = defineEmits(['close', 'update:modelValue']);
+const email = ref('');
+const { subscribe, reset, postSubscribeState, status, loading, Status } =
+  useEmailSubscription();
+
+function close() {
+  reset();
+  email.value = '';
+  emit('close');
+}
+
+function submit() {
+  subscribe(email.value, props.address);
+}
+</script>
+
+<template>
+  <BaseModal :open="open" @close="close">
+    <template #header>
+      <div class="flex flex-row items-center justify-center">
+        <h3>{{ $t('emailSubscription.title') }}</h3>
+      </div>
+    </template>
+    <BaseMessageBlock
+      v-if="postSubscribeState.message.value"
+      :level="postSubscribeState.level.value"
+      class="m-4"
+    >
+      {{ postSubscribeState.message.value }}
+    </BaseMessageBlock>
+    <div v-else class="m-4">
+      {{ $t('emailSubscription.description') }}
+    </div>
+    <form v-if="status !== Status.success" class="m-4" @submit.prevent="submit">
+      <BaseInput
+        v-bind="$attrs"
+        :model-value="email"
+        type="email"
+        :placeholder="'johndoe@gmail.com'"
+        class="!pl-[40px]"
+        required
+        autocomplete="off"
+        @update:model-value="emit('update:modelValue', $event)"
+      >
+        <template #before>
+          <i-ho-mail class="text-[16px]" />
+        </template>
+      </BaseInput>
+
+      <small>
+        You may be asked to sign a transaction to verify wallet ownership
```

Comment:
do we need a translation for this?

Reply:
No, this should say `sign message` instead

Reply:
Oh sorry, yes we should add it from defauls.json but down worry about the translation files these are handles through Crowdin.